pkg/webhook/patch.go

```diff
@@ -607,3 +618,37 @@ func hasInitContainer(pod *corev1.Pod, container *corev1.Container) bool {
 	}
 	return false
 }
+
+func addTerminationGracePeriodSeconds(pod *corev1.Pod, app *v1beta2.SparkApplication) *patchOperation {
+	path := "/spec/terminationGracePeriodSeconds"
+	var gracePeriodSeconds *int64
+
+	if util.IsDriverPod(pod) {
+		gracePeriodSeconds = app.Spec.Driver.TerminationGracePeriodSeconds
+	} else if util.IsExecutorPod(pod) {
+		gracePeriodSeconds = app.Spec.Executor.TerminationGracePeriodSeconds
+	}
+	if gracePeriodSeconds == nil {
+		return nil
+	}
+	return &patchOperation{Op: "add", Path: path, Value: gracePeriodSeconds}
```

Comment:
`Value` should be `* gracePeriodSeconds`.

Reply:
done